src/settings/general-settings.test.tsx

```diff
@@ -23,6 +27,34 @@ describe('General setting', () => {
 	const { defaultI18n } = useI18nStore.getState();
 	const localeArray = localeList(defaultI18n.t);
 
+	test('When there are changes and an error, the discard button is enabled but save button is disabled', async () => {
```

Comment:
the error come from the stored state of zimbraPrefOutOfOfficeReply and not from a user interaction: you are simulating that the wrong value is retrieved from the BE. Is it as intended?

Reply:
In this test I am simulating any error, it does not matter what causes it. If you want I can add one more test to simulate a specific error from a user interaction